misk-actions/src/main/kotlin/misk/web/interceptors/LogRequestResponse.kt

```diff
@@ -34,4 +37,17 @@ annotation class LogRequestResponse(
   val errorBodySampling: Double = 0.0,
   /** which deploy environments will not have request/response logging enabled **/
   val excludedEnvironments: Array<String> = [],
+  /** By default log non-error responses **/
+  val requestLoggingConstraints: RequestLoggingConstraints = RequestLoggingConstraints.ALL,
 )
+
+enum class RequestLoggingConstraints {
```

Comment:
what do you think of naming this `RequestLoggingMode` ?

Reply:
Naming this was hard, I'm happy to rename it to that.